Library/Formula/cowpatty.rb

```diff
@@ -1,13 +1,15 @@
-require 'formula'
-
 class Cowpatty < Formula
-  homepage 'http://www.willhackforsushi.com/?page_id=50'
-  url 'http://www.willhackforsushi.com/code/cowpatty/4.3/cowpatty-4.3.tgz'
-  sha1 '8b7cb2015d0534031827f2f06135bf5cf5929d35'
+  homepage "http://www.willhackforsushi.com/?page_id=50"
+  url "http://www.willhackforsushi.com/code/cowpatty/4.6/cowpatty-4.6.tgz"
+  sha256 "cd3fc113e5052d3ee08ab71aa87edf772d044f760670c73fde5d5581d7803bc2"
+
+  depends_on "openssl"
 
   def install
-    ENV.j1
-    inreplace "Makefile", "/usr/local/", "#{prefix}/"
-    system "make install"
+    system "make", "BINDIR=#{bin}", "install"
+  end
+
+  test do
+    system bin/"cowpatty", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!